common/static/js/src/utility.js

```diff
@@ -42,6 +42,6 @@ window.rewriteStaticLinks = function(content, from, to) {
 // Utility method for replacing absolute URLs
 window.rewriteCdnLinksToStatic = function(content) {
     'use strict';
-    var regex = new RegExp('((https?:)?[/][/](www.)?[-a-zA-Z0-9@:%._+~#=]{2,256}[a-z]{2,6}([-a-zA-Z0-9@:%_+~#?&//=]*)|[a-z-]{14})[/]', 'g');// eslint-disable-line max-len
+    var regex = new RegExp('((https?:)?[/][/](www.)?[-a-zA-Z0-9@:%._+~#=]{2,256}[a-z]{2,6}([-a-zA-Z0-9@:%._+~#?&//=]*)|[a-z-]{14})[/]', 'g');// eslint-disable-line max-len
```

Comment:
maybe if we can get the list of all the characters possible in the course key so that we can add/test all of them in one go.

Reply:
Yes, almost all characters are added except this one ' \ '. I can add this and update the PR.